### DISCUSSION TITLE
How to start timer when not in Kanban view?

### DISCUSSION THREAD

**Discussion by archlinux-fan** [Leantime/leantime#552] [Q&A]
I see an option to start the timer in Kanban view. I don't know how to start time tracking when Table view is selected. Are there other options? It's hard to keep changing views just for tracking.

**Comment by marcelfolaron** [comment]
Good catch. There is currently not an option to start the timer from that view. However it would be trivial to add the 3 dot menu from the kanban view to the left of the title in the table view. Making this a an issue
